src/components/restaurantDetails/RestaurantDetails.js

```diff
@@ -1,9 +1,9 @@
 /* eslint-disable jsx-a11y/img-redundant-alt */
```

Comment:
These components are only used inside restaurant page so they should not be outside